src/components/pages/InstructorNewsFeed/NewsFeedPostModal.js

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react';
+import { Form, Input, Button } from 'antd';
+import axios from 'axios';
+import '../../../styles/InstructorStyles/index.less';
+import { CloseOutlined } from '@ant-design/icons';
+
+const NewsfeedPostModal = ({ setPostOptions }) => {
+  const [formValues, setFormValues] = useState({
+    link: '',
+    description: '',
+    title: '',
+  });
+
+  const handleChange = e => {
+    setFormValues({
+      ...formValues,
+      [e.target.name]: e.target.value,
+    });
+  };
+
+  const token = JSON.parse(localStorage.getItem('okta-token-storage'));
+  const config = {
+    headers: { Authorization: `Bearer ${token.idToken.value}` },
+  };
+
+  const handleSubmit = () => {
+    axios
+      .post('https://coder-heroes-api.herokuapp.com/news', formValues, config)
+      .then(resp => {
+        setPostOptions('newsFeed');
+      })
+      .catch(err => {
+        console.log(err);
```

Comment:
I heard that console.errors are generally better to use than console.logs? Up to you though, just something to consider.

Reply:
changing to console.error(err)